.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 python:
   - "2.7"
   - "3.5"
+cache: pip
```

Comment:
Yeah, dunno. You can [cache pip files](https://docs.travis-ci.com/user/caching/#pip-cache) between builds so they don't have to be built again every time. Not sure if that makes any difference time-wise or if it doesn't screw up stuff.